### ISSUE TITLE
[Bug]: Ars Magica 1.19.3 Release Bugged

### ISSUE THREAD

**Issue by Zelioth** [MinecraftschurliMods/Ars-Magica-Legacy#368]
### Describe the bug

Seems like the update today has a few dependency issues. 

### Steps to Reproduce

install the mod
Pray to someone
Eat bagels 
Have a good time

### Expected behavior

To work

### Screenshots

No screenshots just logs.

### Version

1.19.2-1.2.4

### Forge Version

44.1.5

### Other Mods

Minecraft 1.19.3
Ars Magica 1.2.4
The version shows on forges mod page compatible for 1.19.3 and the ars magica does say it looks for 1.19.3 but doesn't recognize it 

### Log

```shell
https://gist.github.com/Zelioth/ceb8d6b964be98082b5e2c8ee730b5c6
```

**Comment by Zelioth** [comment]
Also I do have GeckoLib Version 1.19.3 4.0.3 Installed

**Comment by Minecraftschurli** [comment]
there are no dependency changes in this update ... we haven't released for 1.19.3 yet because patchouli isn't available for 1.19.3 yet
